requirements/pip.txt

```diff
@@ -32,6 +32,7 @@ celery==3.1.23
 django-celery==3.1.17
 git+https://github.com/pennersr/django-allauth.git@d869aff9#egg=django-allauth
 dnspython==1.11.0
+mercurial==4.2
```

Comment:
We should add this to `requirements/testing.txt` instead, we've run into problems specifying mercurial installs with RTD through our requirements.